ci.sh

```diff
@@ -7,18 +7,22 @@ available() {
 main() {
   set -ex -o pipefail
 
-  if ! available autopep8 && available apt; then
-    apt install -y python3-autopep8
+  if ! available autopep8; then
```

Comment:
I already fixed the sudo thing, it's one of the reasons we have conflicts in this PR